src/classes/project_data.py

```diff
@@ -252,7 +256,24 @@ def _set(self, key, values=None, add=False, partial_update=False, remove=False):
     def new(self):
         """ Try to load default project settings file, will raise error on failure """
         import openshot
-        self._data = self.read_from_file(self.default_project_filepath)
+
+        # Try to load user default project
+        if os.path.exists(self.user_project_filepath):
+            try:
+                self._data = self.read_from_file(self.user_project_filepath)
+            except (FileNotFoundError, PermissionError) as ex:
```

Comment:
The code gets a little weird here, so I should explain:

I originally didn't have the `if` wrapped around this, and was just going to use a `FileNotFoundError` exception raised by `self.read_from_file()` to detect that the file doesn't exist.

But that doesn't work, because there's a bug in the exception code inside `read_from_file()` that loses the exception detail. (I'll file another PR for that.) And even if that bug gets fixed, `read_from_file()` **logs** whenever there's an exception trying to read a file, which I don't want for a file that's _expected_ not to be found most of the time.

So, I couldn't do things that way, and had to add the conditional existence test first.